server/sockets/index.mjs

```diff
@@ -170,7 +170,7 @@ const sockets = ({ gameObjects, httpServer, players, FRAME_IN_MS }) => {
 
           // player selected looting bag and stands next to it
           if (type === "LootingBag" && player.dest === null) {
-            const openLootingBagResult = player.canOpenLootingBag();
+            const openLootingBagResult = player.canInteractWithLootingBag();
 
             if (openLootingBagResult === true) {
```

Comment:
`openLootingBagResult` is used once, use `player.canInteractWithLootingBag()` directly:
```suggestion
            if (player.canInteractWithLootingBag() === true) {
```